tutorials/notebooks/pytorch/export/example_pytorch_export.ipynb

```diff
@@ -43,6 +43,35 @@
       "execution_count": null,
       "outputs": []
     },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "In order to export your quantized model to onnx format, and use it for inference, some additional packages are needed. Notice, this is needed only for models exported to onnx, so this part can be skipped if this is not planned:"
```

Comment:
be consistent with "onnx" vs. "ONNX" throughout the notebook (I prefer the latter)